lib/watchpack.js

```diff
@@ -64,7 +64,9 @@ class Watchpack extends EventEmitter {
 	constructor(options) {
 		super();
 		if (!options) options = {};
-		if (!options.aggregateTimeout) options.aggregateTimeout = 200;
+		if (typeof this.watchOptions.aggregateTimeout !== "number") {
+			this.watchOptions.aggregateTimeout = 200;
```

Comment:
I think you'll want to change watchOptions -> options here.

Reply:
Yes indeed, Thanks for pointing out :rocket: